packages/tracing/src/hubextensions.ts

```diff
@@ -165,12 +164,19 @@ function _startTransaction(
   transactionContext: TransactionContext,
   customSamplingContext?: CustomSamplingContext,
 ): Transaction {
-  const transaction = new Transaction(transactionContext, this);
-  return sample(this, transaction, {
+  const client = this.getClient();
+  const options = (client && client.getOptions()) || {};
```

Comment:
```suggestion
  const options = hub.getClient()?.getOptions() || {};
```

Reply:
Aha, this should have been `this.getClient()...`  #similar-but-different :-/